### ISSUE TITLE
[pages] serve should return a 404 response when no static asset is found.

### ISSUE THREAD

**Issue by GregBrimble** [cloudflare/wrangler2#38]
This obviously messes with frameworks which do a:

```js
const response = await env.ASSETS.fetch()
if (response.ok) return response

// else
return new Response(dynamic)
```

which is pretty common. We should return 404s if they are the asset does not exist.

**Comment by GregBrimble** [comment]
Or better yet, remove serve and do-it-ourselves so we have complete control (and a smaller bundle).

**Comment by GregBrimble** [comment]
Fixed in #105 . We'll hopefully get it released soon :)
